pyramid_zipkin/tween.py

```diff
@@ -8,17 +8,43 @@
 from pyramid_zipkin.request_helper import create_zipkin_attr
 from pyramid_zipkin.request_helper import get_binary_annotations
 
-
-ZipkinSettings = namedtuple('ZipkinSettings', [
+ZipkinTuple = namedtuple('ZipkinSettings', [
```

Comment:
This is confusing. A named tuple with a classname of `ZipkinSettings`, but the class object is called `ZipkinTuple`?